check_sites.py

```diff
@@ -111,10 +115,17 @@ def main():
                 return
         old_checker = RwsCheck(old_sites, etlds, icanns)
         check_sets, subtracted_sets = find_diff_sets(old_checker.load_sets(), rws_checker.load_sets())
-        # TODO: add variable and check for subtracted_sets in case of user 
-        # removing old set from the list
     else:
         check_sets = rws_checker.load_sets()
+        if testing_sets:
+            temp_sets = {}
+            for testing_set in testing_sets:
+                if testing_set not in check_sets:
+                    error_texts.append("There was an error loading the set:\n" + testing_set +
+                        " could not be found in related_website_sets.JSON")
+                else:
+                    temp_sets.update({testing_set: check_sets[testing_set]})
+            check_sets = temp_sets
```

Comment:
Personal preference, but I'd lean toward writing this in two steps: first validate the arguments, then compute the subset of the `check_sets` dict. E.g.:

```suggestion
        if testing_sets:
            absent_primaries = [p for p in testing_sets if p not in check_sets]
            if absent_primaries:
                # print some error and quit
            check_sets = {p: check_sets[p] for p in testing_sets}
```